packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -703,30 +689,14 @@ export class GarbageCollector implements IGarbageCollector {
 			nodesToTombstone: [...tombstoneReadyNodes],
 			nodesToDelete: [] as string[],
 		};
-		switch (this.configs.shouldRunSweep) {
-			case "YES":
-				nodesToDelete.push(...sweepReadyNodes);
-				break;
-			case "ONLY_BLOBS":
-				sweepReadyNodes.forEach((nodeId) => {
-					const nodeType = this.runtime.getNodeType(nodeId);
-					if (nodeType === GCNodeType.Blob) {
-						nodesToDelete.push(nodeId);
-					} else {
-						nodesToTombstone.push(nodeId);
-					}
-				});
-				break;
-			default: // case "NO":
-				nodesToTombstone.push(...sweepReadyNodes);
-				break;
+		if (this.configs.sweepAllowed) {
```

Comment:
Nit: See how it looks to just do a ternary operator when declaring { nodesToTombstone, nodesToDelete } above, it may be easier to read.